services/prompter.js

```diff
@@ -281,10 +286,10 @@ async function Prompter(program, requests) {
           if (password) {
             if (FORMAT_PASSWORD.test(password)) { return true; }
             return '🔓  Your password security is too weak 🔓\n' +
-            ' Please make sure it contains at least:\n' +
-            '    > 8 characters\n' +
-            '    > Upper and lower case letters\n' +
-            '    > Numbers';
+              ' Please make sure it contains at least:\n' +
```

Comment:
[Conventions] Whilst I'm personally strongly in favour of updating code style in every file touched by a PR, I believe we were told to do this in separate cards during our onboarding. It'd be good to validate the convention to follow with @SteveBunlon, @larcin or @VincentMolinie.